readme.md

```diff
@@ -5,28 +5,29 @@ A Laravel package which adds the ability to create "showcases", or view boxes wh
 > Still being developed!
 
 ## Todos:
-    - get to 1.0
-        - ~add the admin section~
-        - ~create components for dropping in the view boxes~
-        - add sort order functionality within displays for trophies
-        - unit tests
-        - show trophies associated with a display on the show form
-        - show displays associated with a trophy on the show form
-        - automate including the assets in the test project
-        - ~add validation rule for checking template files~
-        - finish adding flash messages for CRUD
-        - Vue multiselect for trophy display select
-            - combine with sort order selection
-        - redo nav dropdowns to use scrollbars for large amounts
-    - 1.1
-        - Replace buttons with FontAwesome icons where appropriate
-        - add ability to use a custom auth guard
-        - add image uploader that takes care of uploading images to the CDN
-        - ~add way to mass-assign trophies to a display (instead of one at a time)~
-        - add Trophy Groups
-            - and the ability to assign all Trophy Group members to a Display
-        - add Display Groups
-            - and the ability to assign a new Trophy to all Displays in a group
+
```

Comment:
I actually moved all of this into the Github project https://github.com/BrokerExchange/Showcase/projects/1

I agree that checkbox is better than bullet list for this, but since this is already converted into tasks in the project I think we can just remove it entirely.